src/nvim/os/stdpaths.c

```diff
@@ -131,6 +174,10 @@ char *stdpaths_get_xdg_var(const XDGVarType idx)
     ret = xmemdupz(ret, len >= 2 ? len - 1 : 0);  // Trim trailing slash.
   }
 
+  if ((idx == kXDGDataDirs || idx == kXDGConfigDirs) && ret != NULL) {
+    ret = xdg_remove_duplicate(ret, ENV_SEPSTR);
```

Comment:
on Windows can we support both "/" and "\" ?

Reply:
How is that related to this line?

Reply:
Oh I thought `ENV_SEPSTR` was PATHSEP